examples/voting/ballot.v.py

```diff
@@ -22,6 +22,15 @@
 
 voter_count: public(num)
 chairperson: public(address)
+num_proposals: public(num)
+
+@constant
+def delegated(addr: address) -> bool:
+    return self.voters[addr].delegate != 0x0000000000000000000000000000000000000000
```

Comment:
@DavidKnott is there like a `null_addr` or just `null` (which figures out what the null value is for the given type being compared) so we don't ever have to write something like this? Counting the number of 0's will get tedius lol

Wait, does `0x0000000000000000000000000000000000000000 == 0x0`?

Reply:
I asked somewhere else if it's the unique falsey address, in which case wouldn't need to write it explicitly.

Reply:
But this test does pass as you suggested:
```
def test_zero_address(self):
        self.assertEqual(0x0000000000000000000000000000000000000000, 0x0)
```

Reply:
So does this though:
```
self.assertEqual(0x0000000000000000000000000000000000000000, 0)
```

Reply:
Well, if that's true you can probably write `assert not self.voters[addr].delegate` relying on the "falsey-ness" principle and it should work.

Reply:
@fubuloubu Yup `assert not self.voters[addr].delegate` works

Reply:
Hmm I just got 
```
viper.exceptions.InvalidLiteralException: line 35: Cannot read 0x value with length 3. Expecting 42 (address incl 0x) or 66 (bytes32 incl 0x)
```
when I tried to use 0x0 in place of 0x0000000000000000000000000000000000000000. Maybe there was something wrong with my test case above?